src/kOS.Safe/Encapsulation/Structure.cs

```diff
@@ -255,7 +255,7 @@ public virtual StringValue GetKOSInheritance()
 
         public override string ToString()
         {
-            return "Structure ";
+            return KOSNomenclature.GetKOSName(GetType()) + "()"; // print as the KOSNomenclature string name
```

Comment:
That's a valid point, but then I'd like to figure out another way to indicate that the string value is a structure identifier.  Maybe steal the format structure from `VesselEta`: `"ETA: Apoapsis={0} Periapsis={1} Transition={2}""`.  We might also need to audit other types for the same "has a constructor function" limitation.